css/basic.css

```
@@ -44,6 +45,8 @@ img, iframe {border: none; max-width: 100%}
 
 a {color: var(--c1);  text-decoration:none}
 
+a:visited{color: var(--c1)}
```

I think you mean to use the new color here...

```suggestion
a:visited{color: var(--c6)}
```